src/dev/storybook/run_storybook_cli.ts

```diff
@@ -0,0 +1,67 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { join } from 'path';
+import { run, createFlagError } from '../run';
+import { storybookAliases } from './aliases';
+
+const rootDir = join(__dirname, '..', '..');
```

Comment:
I think this lands you in `kibana/src` which won't work for `x-pack`. Trying to run the infra script results in:
```
 info Loading Storybook: /Users/jason/__projects__/kibana/src/x-pack/legacy/plugins/infra/scripts/storybook.js
ERROR UNHANDLED ERROR
ERROR Error: ENOENT: no such file or directory, chdir '/Users/jason/__projects__/kibana/src/x-pack/legacy/plugins/infra'
```
if I switch this to add one more `..`, it fails with:
```
ERROR UNHANDLED ERROR
ERROR Error: ENOENT: no such file or directory, open '/Users/jason/__projects__/kibana/built_assets/storybook/current.config.js'
error Command failed with exit code 1.
```

Reply:
thx, should work now.